### ISSUE TITLE
Editing a filter on a dashboard causes the question to "stick" to my mouse

### ISSUE THREAD

**Issue by noamtwochairs** [metabase/metabase#11938]
**Describe the bug**
When editing the filter on a dashboard, filter options do not appear on questions until I click on a question. When I click on the question, it becomes "stuck" to my mouse, following it on screen until I exit the filter dialog. Once I exit the question remains stuck or moves off screen, and the only way to revert the chart positioning issue is to cancel the dashboard edit.

**Logs**
When I first click the filter, the error is:
```
backend.js:32 Uncaught Error: Unknown query type: undefined
    at e.value (app-main.bundle.js?1283043d8d6d3679e46b:5)
    at app-main.bundle.js?1283043d8d6d3679e46b:5
    at i (app-main.bundle.js?1283043d8d6d3679e46b:5)
    at e.n.value (app-main.bundle.js?1283043d8d6d3679e46b:5)
    at Object.t.getParameterMappingOptions (app-main.bundle.js?1283043d8d6d3679e46b:5)
    at app-main.bundle.js?1283043d8d6d3679e46b:5
    at vendor.bundle.js?1283043d8d6d3679e46b:1
    at vendor.bundle.js?1283043d8d6d3679e46b:1
    at vendor.bundle.js?1283043d8d6d3679e46b:1
    at vendor.bundle.js?1283043d8d6d3679e46b:1
```

Then when I click the chart to show the filter edit UI:
```
backend.js:32 Uncaught TypeError: Cannot read property 'getHostNode' of null
    at Object.getHostNode (vendor.bundle.js?1283043d8d6d3679e46b:1)
    at c.getHostNode (vendor.bundle.js?1283043d8d6d3679e46b:1)
    at Object.getHostNode (vendor.bundle.js?1283043d8d6d3679e46b:1)
    at Object.updateChildren (vendor.bundle.js?1283043d8d6d3679e46b:1)
    at Z._reconcilerUpdateChildren (vendor.bundle.js?1283043d8d6d3679e46b:1)
    at Z._updateChildren (vendor.bundle.js?1283043d8d6d3679e46b:1)
    at Z.updateChildren (vendor.bundle.js?1283043d8d6d3679e46b:1)
    at Z._updateDOMChildren (vendor.bundle.js?1283043d8d6d3679e46b:1)
    at Z.updateComponent (vendor.bundle.js?1283043d8d6d3679e46b:1)
    at Z.receiveComponent (vendor.bundle.js?1283043d8d6d3679e46b:1)
```

and when I click "cancel" to exit the dashboard edit mode, the trace is:
```
Uncaught Invariant Violation: Minified React error #136; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=136&args[]=Text for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at e.exports (https://data.twochairs.com/app/dist/vendor.bundle.js?1283043d8d6d3679e46b:1:285930)
    at c.updateComponent (https://data.twochairs.com/app/dist/vendor.bundle.js?1283043d8d6d3679e46b:1:2220601)
    at c.receiveComponent (https://data.twochairs.com/app/dist/vendor.bundle.js?1283043d8d6d3679e46b:1:2220200)
    at Object.receiveComponent (https://data.twochairs.com/app/dist/vendor.bundle.js?1283043d8d6d3679e46b:1:704155)
    at Object.receiveComponent (chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/backend.js:32:13957)
    at Object.e.<computed> [as receiveComponent] (chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/backend.js:32:10174)
    at Object.updateChildren (https://data.twochairs.com/app/dist/vendor.bundle.js?1283043d8d6d3679e46b:1:2215400)
    at Z._reconcilerUpdateChildren (https://data.twochairs.com/app/dist/vendor.bundle.js?1283043d8d6d3679e46b:1:2212780)
    at Z._updateChildren (https://data.twochairs.com/app/dist/vendor.bundle.js?1283043d8d6d3679e46b:1:2213717)
    at Z.updateChildren (https://data.twochairs.com/app/dist/vendor.bundle.js?1283043d8d6d3679e46b:1:2213615)
```

**To Reproduce**
1. Go to a dashboard
2. Click "edit this dashboard"
3. Edit a filter
4. Click on a question
5. Have the question follow your mouse on screen until you refresh. 

**Expected behavior**
I would expect to be able to edit the filters on the question.

**Screenshots**
![metabase_chart_edit_issue](https://user-images.githubusercontent.com/44379107/74798885-dcfd4280-5283-11ea-8db0-bbac86f2513e.gif)

**Information about your Metabase Installation:**

You can get this information by going to Admin -> Troubleshooting.

```json
{
  "browser-info": {
    "language": "en-US",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.5+10",
    "java.vendor": "AdoptOpenJDK",
    "java.vendor.url": "https://adoptopenjdk.net/",
    "java.version": "11.0.5",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.5+10",
    "os.name": "Linux",
    "os.version": "4.14.138+",
    "user.language": "en",
    "user.timezone": "GMT"
  },
  "metabase-info": {
    "databases": [
      "postgres"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "11.6"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.2.8"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2020-02-04",
      "tag": "v0.34.2",
      "branch": "release-0.34.x",
      "hash": "dab738c"
    },
    "settings": {
      "report-timezone": null
    }
  }
}
```

**Severity**
It's preventing us from updating dashboards our users and my colleagues see every day, so the impact is medium-high for our productivity.

**Comment by dker92** [comment]
Yeah I got the same problem with the latest version of metabase

**Comment by flamber** [comment]
Duplicate of #11927
It has already been fixed and will be part of 0.34.3 - I don't know an exact release date, but very soon.
The only workaround is remove text boxes or downgrade to 0.34.0 (remember to backup first!)
